File: CPE/CPE49/03-UVA10035.py
while True:
    x, y = map(int, input().split())
    if x == y == 0:
        break
    ans = 0
    carry = 0
    while x or y:
        if x % 10 + y % 10 + carry > 9:
            ans += 1
            carry = 1
        else:
            carry = 0
        x //= 10
        y //= 10
    print((str(ans) if ans else "No") + " carry operation" + ("s." if ans > 1 else "."))

File: CPE/CPE49/03-UVA10035.cpp
#include <bits/stdc++.h>
using namespace std;
using LL = long long;
#define endl '\n'

int main() {
    ios::sync_with_stdio(false); cin.tie(nullptr); cout.tie(nullptr);
    LL x, y;
    int ans, carry;
    while (cin >> x >> y && (x || y)) {
        ans = 0;
        carry = 0;
        while (x || y) {
            if (x % 10 + y % 10 + carry > 9) {
                ans++;
                carry = 1;
            } else {
                carry = 0;
            }
            x /= 10;
            y /= 10;
        }
        cout << (ans ? to_string(ans) : "No") << " carry operation" << (ans > 1 ? "s." : ".") << endl;
    }
    return 0;
}